css/style.css

```diff
@@ -0,0 +1,460 @@
+/* Defaults */
+* {
+  box-sizing: border-box;
+
+  /* overflow-x: hidden; */
+}
+
```

Comment:
Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /\* `.nav { color: red;}` \*/  
:heavy_check_mark: **comments about code** /\*Navigation \*/  
Comments about the code are okay and should only be tagged as `OPTIONAL` if it's unnecessary and please provide details about why it is unnecessary.

Reply:
Thank you. I have removed inline comments